charts/legend-deployment/templates/deployment.yaml

```diff
@@ -119,12 +119,17 @@ spec:
           - name: {{ $volumeMount.name }}
             mountPath: {{ $volumeMount.mountPath }}
           {{- end }}
+          {{- range $externalSecret := .externalSecrets }}
```

Comment:
Legend deployment will also need to support secrets through environment variable, i.e. without mounting. Similar to how we have a `secretRefEnvironment` property in the values file :)